### ISSUE TITLE
Sort out how Aqua will scan *-prod namespaces

### ISSUE THREAD

**Issue by jleach** [BCDevOps/OpenShift4-RollOut#363]
Due to resource constraints we don't want to scan all images in the cluster at any scheduled interval. Rather, we want to periodically scan `*-prod` namespaces and let teams scan images are they are build (part of CICD pipeline) or when new images are added to the cluster.

**Comment by mitovskaol** [comment]
@NickCorcoran If you already have a clear idea of how AQUA will scan prod namespaces, please document it here and close the ticket. If not, please book a meeting with the team to brainstorm it together.

**Comment by NickCorcoran** [comment]
Agreed some brainstorming is needed, as well as standardization on expected image naming.  Many projects are using -tools images w/ latest label for builds to all environments.  I'll set something up for this.
